catroid/src/main/java/org/catrobat/catroid/pocketmusic/ui/NoteView.java

```diff
@@ -30,25 +30,33 @@
 import android.widget.ImageView;
 
 import org.catrobat.catroid.R;
+import org.catrobat.catroid.pocketmusic.note.NoteLength;
+import org.catrobat.catroid.pocketmusic.note.trackgrid.GridRowPosition;
+
+import java.util.List;
 
 public class NoteView extends ImageView implements View.OnClickListener {
 
 	private static final int HIDDEN = 0;
 	private static final int FULL_VISIBLE = 255;
+	private final List<GridRowPosition> gridRowPositions;
+	private final int index;
```

Comment:
please rename the two new member variables - it is not clear what they are doing here